README.md

```diff
@@ -1,56 +1,54 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Leaderboard
 
+> "To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. A simple website that allows user to create a To-do list.
```

Comment:
- [ ] Kindly update the readme file and make sure everything represent the current project. We all know that the Todo list and the leaderboard project are not the same.